src/modules/common/components/ScrollingSearchBox.js

```diff
@@ -52,8 +42,6 @@ export class ScrollingSearchBox extends React.PureComponent {
 
   onSearchInputTextChange = value => this.onFilterTextChange(value)
```

Comment:
duplicate right here :D
You could even remove both of these functions and just pass the function from the props.